scripts/createPackage.js

```diff
@@ -62,6 +62,12 @@ var platformOptions = {
     platform: 'linux',
     arch: 'armv7l',
     fpm: ['--architecture', 'armhf']
+  },
+  arm64: {
```

Comment:
Maybe this could be renamed to `linuxArm64`, and the other target to `linuxX64`?

Reply:
> Maybe this could be renamed to `linuxArm64`, and the other target to `linuxX64`?

Do we need consider other platform with arm64 architecture?  Maybe the furture mac / windows  platform compiler will auto handle this?

Reply:
The Mac ARM build should share the same binary with the x86 one, but if we wanted to add a Windows ARM build, I think it would be separate. And either way, right now we're assuming "arm64" means "Linux", which isn't right.

Reply:
> The Mac ARM build should share the same binary with the x86 one, but if we wanted to add a Windows ARM build, I think it would be separate. And either way, right now we're assuming "arm64" means "Linux", which isn't right.

Yes, we should consider windows ARM build.  I will update it to `linuxArm64` for sperate other system support arm in the furture.

I prefer other linux target use `linuxAmd64`.  AMD64 more popular present.

Reply:
> Maybe this could be renamed to `linuxArm64`, and the other target to `linuxX64`?
If we need also handle linux i386 ,the it renamed to linuxX64 / linuxAmd64 will make sense.
Otherwise the original linux target keep the same name.

Reply:
> I prefer other linux target use linuxAmd64. AMD64 more popular present.

Sounds fine to me.


> If we need also handle linux i386 ,the it renamed to linuxX64 / linuxAmd64 will make sense.

Chromium dropped i386 support a while ago, so that won't be needed.

Reply:
done